pkg/packagekit/kpatch.jsx

```diff
@@ -259,7 +259,7 @@
         const body = <Form><Checkbox id="apply-kpatch"
                                isChecked={this.state.applyCheckbox}
                                label={_("Apply kernel live patches")}
-                               onChange={checked => this.setState({ applyCheckbox: checked })}
+                               onChange={(_event, checked) => this.setState({ applyCheckbox: checked })}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18721-20230426-094458-1180deae-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)